k8s/record-deployment-job.yaml.j2

```diff
@@ -0,0 +1,49 @@
+kind: Job
+apiVersion: batch/v1
+metadata:
+  name: developerportal-{{ APP_NAME }}-record-deployment
+spec:
+  template:
+    metadata:
+      name: developerportal-{{ APP_NAME }}-record-deployment
+    spec:
+      restartPolicy: Never
+      containers:
+        - name: developerportal-{{ APP_NAME }}-record-deployment
+          image: {{ APP_IMAGE }}:{{ APP_IMAGE_TAG }}
+          imagePullPolicy: {{ APP_IMAGE_PULL_POLICY }}
+          resources:
+            requests:
+              cpu: 100m
+              memory: 256Mi
+            limits:
+              cpu: 500m
+              memory: 1Gi
```

Comment:
I don't really think we need to give it that high of a limit, I am happy to say CPU `256m` and memory `512Mi`